### DISCUSSION TITLE
Error on Start - No CUDA-capable driver

### DISCUSSION THREAD

**Discussion by taz4535** [opendatacam/opendatacam#509] [Q&A]
I've followed the instructions as per ReadMe, im running this on docker desktop, and completed the settings for this including drivers, etc. From what i've seen it should be fairly straight forward especially within docker to get this up and running as it comes with sample settings and video? Any help would be amazing.

On start it compiles and logs appear fine, however do get this error about db connection timed out.

Process YOLO initialized
> Ready on http://localhost:8080
> Ready on http://172.17.0.2:8080
MongoServerSelectionError: connection timed out
    at Timeout._onTimeout (/var/local/opendatacam/node_modules/mongodb/lib/core/sdam/topology.js:430:30)

Perhaps, also a follow on from the above error, when opening the website the below is a looping error I find in the logs

CUDA Error: no CUDA-capable device is detected: Bad file descriptor
darknet: ./src/utils.c:326: error: Assertion `0' failed.
Restart YOLO

PC Info: NVIDIA GeForce GTX 980, Win 10

**Comment by munsterlander** [comment]
Interesting error.  This almost seems like the Mongo issue where it installed the latest version and not the version that ODC needs.  Did you create your own Docker container or use the one compose file provided?

**Reply by taz4535** [reply]
Used: docker pull opendatacam/opendatacam:v3.0.2-desktop
then clicked run from the image to get the app, open in browser gets me to the DB timeout issue. I assume the compose file is already built in?

**Reply by vsaw** [reply]
Try using 'mongo:4.4.8' in the docker compose file. Maybe that helps 🤷‍♂️
